ddht/v5_1/alexandria/client.py

```diff
@@ -625,6 +636,12 @@ async def _feed_talk_responses(self) -> None:
                 except DecodingError:
                     pass
                 else:
+                    if message.type != AlexandriaMessageType.RESPONSE:
+                        self.logger.warning(
```

Comment:
I think this should be `DEBUG`.  Otherwise, a malfunctioning (or malicious) client will cause log spam.